cypress/obj/loginPage.js

```diff
@@ -0,0 +1,88 @@
+class LoginPage 
+{
+    visit()
+    {
+        cy.visit('/');
+    }
+    getUserValidationIndicator()
+    {
+        return cy.get('.Select.not-valid');
+    }
+
+    getSubmitButton()
+    {
+        return cy.get('[type="submit"]');
+    }
+
+    getSpecificUserFromDropDown(userName)
+    {
+        return cy.get('[aria-label="' + userName + '"]');
+    }
+
+    openUpUserDropDown()
```

Comment:
This function would probably be better called "get" instead "open", because you do not actually click it here in the definition, or you could add "click()" at the end :)